src/org/opendatakit/briefcase/util/FormCacheble.java

```diff
@@ -1,5 +1,6 @@
 package org.opendatakit.briefcase.util;
```

Comment:
`FormCacheable` would be a better spelling for this class.

See this, for example:
https://docs.oracle.com/javaee/6/api/javax/persistence/Cacheable.html

Reply:
It's French and it's pronounced "casheblé"

I'm kidding. Let's fix that typo!